site/content/kapp-controller/docs/latest/packaging-tutorial.md

```diff
@@ -0,0 +1,427 @@
+## Package Management Tutorial
```

Comment:
So I am not necessarily against having the tutorial here, but I will call out that the website will now have four different walkthroughs:
* One for the App CR
* One for Package Consumption
* One for Package Authors
* The katacoda tutorial 

I think I would prefer to not have the whole tutorial as a separate page as part of the initial roll out. It would be nice to consolidate all the walkthroughs via this tutorial and eventually remove all the existing ones.

Reply:
Maybe what we could do is mention in the Package Author/Package consumption workflows that this tutorial exists if someone does not want to set up a local environment. It might also be nice to more directly link to this tutorial somewhere on the home page.